README.md

```diff
@@ -1,7 +1,7 @@
 # FixtureFactory
 
 FixtureFactory is an attempt to merge concepts from [Rails Fixtures](http://api.rubyonrails.org/v5.2.0/classes/ActiveRecord/FixtureSet.html),
-[Hermes Builders](https://github.com/plataformatec/hermes/blob/master/lib/hermes/builders.rb),
+[Hermes](https://web.archive.org/web/20180611003053/https://github.com/plataformatec/hermes) builders,
```

Comment:
What if we linked to the gem instead? https://rubygems.org/gems/hermes

Reply:
We can just remove this link and Hermes from the list.

Reply:
Yeah, if it doesn't add anything meaningful to the description then let's remove it. I've pushed an update.